### PR TITLE
Add sunday on calendar component

### PR DESCRIPTION
**Pull request by oscarmcm**

### CODE REVIEW

File: app/components/calendar/Calendar.jsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ class Calendar extends BaseComponent {
   calculateWeek(date) {
     const self = this;
     const dateOfWeek = date.clone();
-    const mondayKey = dateOfWeek.startOf('isoWeek').format('YYYY-MM-DD');
+    const mondayKey = dateOfWeek.isoWeekday(1).format('YYYY-MM-DD');
     if (self.props.weeksMap[mondayKey]) return self.props.weeksMap[mondayKey].number_week;
     return "";
```
Comment (paulomcnally) [inline app/components/calendar/Calendar.jsx]: Comillas simples.

File: app/components/calendar/Calendar.jsx [hunk 2]
```diff
@@ -51,7 +51,17 @@ class Calendar extends BaseComponent {
...
+          key={date.toString()}
+          date={date.clone()}
+          month={this.state.month}
+          week={(date) => this.calculateWeek(date)}
+          select={this.select.bind(this)}
+          selected={this.state.selected}
+          hasWeeks={this.props.hasWeeks}
+        />
+      );
       date.add(1, "w");
```
Comment (paulomcnally) [inline app/components/calendar/Calendar.jsx]: Comillas simples

File: app/components/calendar/Week.jsx [hunk 3]
```diff
@@ -13,18 +13,18 @@ class Week extends BaseComponent {
     const month = self.props.month;
     const week = self.props.week;
 
-    for (let i = 0; i < 7; i++) {
+    for (let i = 0; i <= 7; i++) {
       const day = {
         name: date.format("dd").substring(0, 1),
```
Comment (paulomcnally) [inline app/components/calendar/Week.jsx]: Comilla simples

**Comment by oscarmcm** [comment]
@paulomcnally I've pushed a new commit that change to single quote

**Pull request merged**